tools/Cargo.toml

```diff
@@ -61,6 +61,8 @@ tree-sitter = "0.20.9"
 tree-sitter-python = "0.20.2"
 tree-sitter-rust = "0.20.3"
 tree-sitter-typescript = "0.20.2"
+tree-sitter-java = "0.20.2"
+tree-sitter-kotlin = { git = "https://github.com/nicolas-guichard/tree-sitter-kotlin.git", branch = "field-names" }
```

Comment:
TODO: replace with https://github.com/fwcd/tree-sitter-kotlin branch=main once fwcd/tree-sitter-kotlin/pull/106 is merged. Then with a proper release once available.